tests/model_serving/model_server/ovms/model_mesh/test_tensorflow.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from tests.model_serving.model_server.utils import verify_inference_response
+from utilities.constants import (
+    ModelStoragePath,
+    Protocols,
+    ModelInferenceRuntime,
+)
+from utilities.inference_utils import Inference
+
+
+pytestmark = [pytest.mark.modelmesh]
+
+
+@pytest.mark.parametrize(
+    "ns_with_modelmesh_enabled, http_s3_ovms_model_mesh_serving_runtime,"
+    " http_s3_tensorflow_model_mesh_inference_service",
+    [
+        pytest.param(
+            {"name": "model-mesh-tensorflow"},
+            {"enable-external-route": True},
+            {"model-path": ModelStoragePath.TENSORFLOW_MODEL},
+        )
+    ],
+    indirect=True,
+)
+class TestTensorflowModelMesh:
```

Comment:
Shouldn't you check that modelmesh is enabled here or in `ns_with_modelmesh_enabled`?

Reply:
logic is changed in following pr; all tests which are marked as model-mesh will have the check
added in model_ns to enable model mesh if disabled
fixed in coming pr